### PR TITLE
fix gear icon in fim table

### PR DESCRIPTION
**Pull request by eze9252**
Hi team, this resolve:

- fix position gear icon im fim table

### CODE REVIEW

**Comment by Machi3mfl** [comment]
Fixed in small resolutions.

Screen Shot 2021-04-29 at 15 32 35

**Review by frankeros — CHANGES_REQUESTED** [review]

File: SplunkAppForWazuh/appserver/static/js/controllers/agents/fim/agents-fim.html [hunk 1]
```diff
@@ -261,7 +261,7 @@
                 </button>
               </div>
               <div layout="row" ng-if="agent" class="wz-margin-top-10 wz-margin-bottom-40-inv">
-                <wazuh-table custom-columns="true" flex path="'/syscheck/' + agent.id" row-sizes="[12,10,8]"
+                <wazuh-table id="agents-fim-table" custom-columns="true" flex path="'/syscheck/' + agent.id" row-sizes="[12,10,8]"
```
[the remark this answers is not in the document]
Comment (Machi3mfl) [reply SplunkAppForWazuh/appserver/static/js/controllers/agents/fim/agents-fim.html]: Fixed, changed id property for class property.

**Comment by Desvelao** [comment]
Tested ✅ 

Good job! 

image

**Review by Desvelao — APPROVED** [review]
LGTM!

**Review by frankeros — APPROVED** [review]
LGTM!

**Pull request merged**